validations/src/validations.rs

```diff
@@ -129,14 +162,20 @@ pub fn validate_consensus(
     }
 }
 
+/// Function to validate a value transfer transaction
```

Comment:
(***) _Suggestion_: this kind of comment add almost no insight on what the function does and transmits the same information as the name of the function. A better way would be to describe what does it mean for a value-transfer transaction to be valid and in which situations that might fail

Reply:
Created issue #550